docs/Manual/03 Schedules.md

```diff
@@ -62,10 +63,11 @@ for k in range(11):
             C[i, j] += A[i, k] * B[k, j]
 ```
 
-Some orders are not allowed. Describing the restrictions in full requires concepts that have not yet been introduced, so we merely mention the restrictions here and explain them in detail later on. The restrictions are:
-1. The *inner dimension* created by a `split` transformation (see below) always comes after its corresponding *outer dimension*.
+However, some orders are not allowed. Comprehending these restrictions and their logical explanation at this stage requires understanding concepts that are yet to be introduced. Therefore, we are stating these restrictions here and will discuss them later in the upcoming sections. The restrictions are: 
```

Comment:
"Describing these restrictions in full will require concepts that are yet to be introduced."